MODULE.bazel

```diff
@@ -886,3 +891,98 @@ mainnet_versions(
     name = "mainnet_versions",
     path = "//:mainnet-subnet-revisions.json",
 )
+
+# Set up hermetic cc toolchains for binaries and canisters
+toolchains = use_extension("@hermetic_cc_toolchain//toolchain:ext.bzl", "toolchains")
+toolchains.extra_settings(settings = ["//bazel:use_hermetic_cc"])
+use_repo(toolchains, "zig_sdk")
+
+register_toolchains(
+    # if no `--platform` is specified, these toolchains will be used for
+    # (linux,darwin,windows)x(amd64,arm64)
+    "@zig_sdk//toolchain:linux_amd64_gnu.2.28",
+
+    # wasm toolchains
+    "@zig_sdk//toolchain:none_wasm",
+
+    # These toolchains are only registered locally.
+    dev_dependency = True,
+)
+
+# Build libfuzzer to ensure compatibility with our toolchain, and so that it
+# can be optionally linked by the fuzzers.
+http_archive(
+    name = "libfuzzer",
```

Comment:
out of curiosity, how was it provided before?

Reply:
Previously from [here](https://github.com/dfinity/ic/blob/f0c82237ae16745ac54dd3838b3f91ce32a6bc52/bazel/fuzz_testing.bzl#L57) and just below. There's some more context in this thread: https://github.com/dfinity/ic/pull/3508#discussion_r1979503363